packages/checkout/sdk/src/types/smartCheckout.ts

```diff
@@ -319,7 +319,7 @@ export type ERC20Item = {
   /**  The type to indicate this is an ERC20 item. */
   type: ItemType.ERC20;
   /** The contract address of the ERC20. */
-  contractAddress: string;
+  tokenAddress: string;
```

Comment:
the @property and the comment above needs updating to say token address and tokenAddress

Reply:
Fixed